app/main.py

```diff
@@ -14,0 +1,13 @@
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+
+    Actor.objects.create(first_name="Emma", last_name="Watson")
+    Actor.objects.create(first_name="Daniel", last_name="Radcliffe")
+    print(Actor.objects.all())
+    Actor.objects.update(2, "Daniel", "Radcliffe")
```

Comment:
Ensure that the `id` field in the database is set to auto-increment. This will allow the `update` method to correctly identify and update the actor with the given `id`.